packages/react-core/src/demos/examples/Tabs/TabsAndTable.tsx

```diff
@@ -135,15 +136,45 @@ export const TablesAndTabs = () => {
     }
   ];
 
+  const firstActionRef = React.useRef<HTMLButtonElement>(null);
+
+  const handleClick = (event: React.MouseEvent, ActionsToggleProps: CustomActionsToggleProps) => {
```

Comment:
I'm not sure if I know quite what you mean by not apparent? Is the potential confusion just coming from the names that are a bit too generic?

I could definitely say that the KeyDown handler is to enable keyboard navigation of the toggle, and change the names so that it's more clear what these handlers are for, but I'm not sure what there really is to explain about the Click handler.

Reply:
I guess I mean - why are they needed? 
Users don't ususally need to wire up keydown and click handling for our basic dropdowns, so this won't be what they expect.

Reply:
I added a couple of comments and made the handler names more descriptive, WDYT?